test/integration/configurations/nighthawk_http_origin.yaml

```diff
@@ -29,7 +29,19 @@ static_resources:
               - "*"
           http_filters:
           - name: time-tracking
+            typed_config:
```

Comment:
Does it still work if we delete `name: time-tracking`? I heard but haven't confirmed that the type under `typed_config` type takes precedence over `name`. If that's true, we could avoid confusion by deleting `name` throughout.

Reply:
(Just saw your note in the description that a full migration might only be possible later.)

Reply:
While it might work to remove the name, I'm not really sure what that would gain us.